src/bash/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
+!Makefile
```

Comment:
Why you need unignore? Who ignore it at first place?

Reply:
The first line, `*` ignores everything, then we un-ignore the Makefile here

Reply:
Why ignore everything? That will make things complex.

Reply:
Because we only want to commit the Makefile. We don't care about the generated files, and if the generated files change, we will need to update this. This way, we never need to update it unless we need to add any files.